00-demo.md

```diff
@@ -0,0 +1,37 @@
+# Demo Chapter
+
+Here's some code that'll run in a notebook and also run in CI.
+
+```python
+print("Hello world")
+```
+
+## How to Use the Setup
+
+You can write chapters for the course using standard markdown files in a variety of ways. Here are a few approaches that we've found useful. Before getting started, you need to set up a Python environment and install the Python requirements. Once you have Python set up, run `pip install -r requirements.txt` from the root directory to install all the requirements.
+
+### The Bare-Bones Approach
+
+1. Use a text editor of your choice to modify the markdown. Any fenced code block starting with the `python` language identifier will be executed when you convert the markdown to html.
+2. Run `make [your markdown document].ipynb.html` from the command line to execute the notebook and convert it to html.
+
+### The Interactive Approach
+
+Recompiling the document with `make` every time you make a small change can be cumbersome. You can get an interactive environment using Jupyter Lab.
+
+1. Run `jupyter lab` from the command line to start a new Jupyter Lab server.
+2. Once the server has loaded (it can take a few seconds), right-click on the markdown file you want to modify, hover your mouse over "Open With", and select "Notebook".
```

Comment:
When I ran `jupyter lab`, right-clicked on the markdown file and hovered over "Open With", I don't see the option to select "Notebook". Do you know why? I've installed Jupyter Lab and Notebook..

![Screenshot 2024-02-13 at 3 08 09 PM](https://github.com/datatrail-jhu/python/assets/50791792/e3ecde7e-95a8-4019-ae1b-c8a03a2cdfac)

Reply:
Just to check, did you run `pip install -r requirements.txt` before starting the server? If yes, could you share the output of `pip freeze`?

Reply:
Interesting, it looks like you may also have to shut down the server, execute `jupyter serverextension enable jupytext`, then restart the server. More information [here](https://jupytext.readthedocs.io/en/latest/install.html#jupytext-s-contents-manager).

Reply:
Okay cool! I managed to solve the issue. BTW, the jupytext doc has a typo. This command actually ran and fixed the issue:  `jupyter server extension enable jupytext`

Reply:
Great, I've added a line about enabling the extension.